packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: flutter_style_todos
```

Comment:
Because of: https://github.com/flutter/flutter/pull/24381/files#diff-afa141921d14a02769e479c13b0b775cR29